<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Core::Internal::ExternalToolConfig</class>
 <widget class="QWidget" name="Core::Internal::ExternalToolConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTreeWidget" name="toolTree">
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="infoWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>10</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="descriptionLabel">
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="description"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="executableLabel">
        <property name="text">
         <string>Executable:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="Utils::PathChooser" name="executable">
        <property name="expectedKind">
         <enum>Utils::PathChooser::ExistingCommand</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="argumentsLabel">
        <property name="text">
         <string>Arguments:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="arguments"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="workingDirectoryLabel">
        <property name="text">
         <string>Working directory:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="Utils::PathChooser" name="workingDirectory"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="outputLabel">
        <property name="text">
         <string>Output:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="outputBehavior">
        <item>
         <property name="text">
          <string>Ignore</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Show in pane</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Replace selection</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Reload document</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="errorOutputLabel">
        <property name="text">
         <string>Error output:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="errorOutputBehavior">
        <item>
         <property name="text">
          <string>Ignore</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Show in pane</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Replace selection</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Reload document</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="inputCheckbox">
        <property name="text">
         <string>Input:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPlainTextEdit" name="inputText">
        <property name="lineWrapMode">
         <enum>QPlainTextEdit::NoWrap</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
